app/Resources/translations/default/messages.xlf

```diff
@@ -2691,6 +2676,21 @@
         <target>This action will permanently remove the module from the server. Are you sure you want to do this?</target>
         <note>Line: 4410</note>
       </trans-unit>
+      <trans-unit id="bcfaccebf745acfd5e75351095a5394a">
```

Comment:
Hello there! It must be localized in _Admin.Actions_, thanks.

Reply:
It is already in `AdminActions.xlf` but it appears in this file as well because in some places the domain is not specified (in this case it's in `AdminController.php` but maybe there is some in other places).

We need to keep it in `messages.xlf` if we want it to be translated everywhere. Of course the better way would be to fix it in the controller, but I'm not sure it's the only place and if it should be done in this PR.

As you said we should clean all the messages contained in this file, but maybe it should be done in 177. @eternoendless what do you think?